app/src/main/res/values/colors.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="colorPrimary">#3F51B5</color>
+    <color name="colorPrimaryDark">#303F9F</color>
+    <color name="colorAccent">#FF4081</color>
+    <color name="colorTitle">#000</color>
+    <color name="backgroundWhite">#FFFFFF</color>
```

Comment:
For new color resources let's use "_" to separate the words: background_white (This could be called white, so it matches for everthing, text fonts, backgrounds, etc.)